var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -33,27 +33,37 @@ class Sundials(Package):
     homepage = "http://computation.llnl.gov/casc/sundials/"
     url = "http://computation.llnl.gov/projects/sundials-suite-nonlinear-differential-algebraic-equation-solvers/download/sundials-2.6.2.tar.gz"
 
+    version('2.7.0', 'c304631b9bc82877d7b0e9f4d4fd94d3')
     version('2.6.2', '3deeb0ede9f514184c6bd83ecab77d95')
 
-    variant('mpi',     default=True,  description='Enable MPI support')
+    variant('mpi',     default=True,
+            description='Enable MPI parallelism')
     variant('lapack',  default=True,
-            description='Build with external BLAS/LAPACK libraries')
+            description='Use external BLAS/LAPACK libraries')
     variant('klu',     default=False,
-            description='Build with SuiteSparse KLU libraries')
+            description='Enable KLU sparse, direct solver')
     variant('superlu', default=False,
-            description='Build with SuperLU_MT libraries')
-    variant('openmp',  default=False, description='Enable OpenMP support')
+            description='Enable SuperLU_MT sparse, direct solver')
+    variant('openmp',  default=False,
+            description='Enable OpenMP parallelism')
     variant('pthread', default=True,
-            description='Enable POSIX threads support')
+            description='Enable Pthreads parallelism')
+    variant('hypre', default=False,
+            description='Enable Hypre parallel vector for MPI parallelism')
 
-    depends_on('cmake', type='build')
+    depends_on('cmake',              type='build')
     depends_on('mpi',                when='+mpi')
+    depends_on('mpi',                when='@2.7:+hypre')
+    depends_on('hypre',              when='@2.7:+hypre')
     depends_on('blas',               when='+lapack')
     depends_on('lapack',             when='+lapack')
     depends_on('suite-sparse',       when='+klu')
     depends_on('superlu-mt+openmp',  when='+superlu+openmp')
     depends_on('superlu-mt+pthread', when='+superlu+pthread')
```

Comment:
Again, are these style issues really what we should be spending our time on. I am sure there are things I am not a "huge fan" of that maybe others do. I just don't wanna spend time on these stylistic issues. If there is something I've indeed broken in Spack style guides, I am amenable to correct. Otherwise, lets please focus more on functionality.